stable/sumologic-fluentd/values.yaml

```diff
@@ -30,9 +30,9 @@ sumologic:
   ## in the container (Default "/fluentd/conf.d/user")
   fluentdUserConfigDir: ""
 
-  ## How frequently to push logs to SumoLogic (default 5s)
+  ## How frequently to push logs to SumoLogic (default 30s)
   ## ref: https://github.com/SumoLogic/fluentd-kubernetes-sumologic#options
-  # flushInterval: 5
+  flushInterval: 5
```

Comment:
It notes the default is 30s but uncomments to set it to 5s. Can we get this resolved?

Reply:
I update that to 30s since that's the upstream default; I can move it back if we want it to mean default for the chart